app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,50 @@
+#
+# frozen_string_literal: true
+
+class AuthorsController < ApplicationController
+  def index
+    @authors = Author.all
+  end
+
+  def show
+    @author = Author.find(params[:id])
```

Comment:
Try to use before_action or before_filter to set Author in Show, Edit and Destroy